### PR TITLE
Add media query for mobile view. [#64]

### PR DESCRIPTION
**Pull request by SinghShreyansh**
I updated _sass/all.scss/
Before : 
Screenshot (195)

After : 
Screenshot (196)

Hope this will solve the issue 👍

### CODE REVIEW

**Comment by yyjlincoln** [comment]
Ref #64

**Review by yyjlincoln — APPROVED** [review]
LGTM

**Review by anubhav06 — APPROVED** [review]
LGTM!

**Review by ditsuke — APPROVED** [review]
LGTM! Thanks

**Comment by ditsuke** [comment]
3 approvals - merging this in. Thanks again and good work!

**Comment by ditsuke** [comment]
@SinghShreyansh can you open another PR for centering the elements? This one should be limited to the media query.

File: Gemfile.lock [hunk 1]
```diff
@@ -70,4 +80,4 @@ DEPENDENCIES
   webrick (~> 1.7)
 
 BUNDLED WITH
-   2.3.4
+   2.3.5
```
Comment (yyjlincoln) [inline Gemfile.lock:83]: Is there a reason for changing the `Gemfile.lock`? Please try and minimise changes to other files & make sure that you don't change other stuff unnecessarily unless it's the purpose of the commit. Thanks!

**Review by yyjlincoln — CHANGES_REQUESTED** [review]
A small thing - please check the comments

**Comment by ditsuke** [comment]
@SinghShreyansh I'm reverting 82493af and beyond then merging this PR. Feel free to open another PR for these changes and I'll be happy to merge after approval.

File: Gemfile.lock (hunk 1 above)
Comment (ditsuke) [reply Gemfile.lock]: Changes reverted with 6c64fd3

**Pull request merged**